test/boostTest.cpp

```diff
@@ -279,7 +279,6 @@ test_suite* init_unit_test_suite( int /*argc*/, char* /*argv*/[] )
 			"ABIDecoderTest",
 			"ABIEncoderTest",
 			"SolidityAuctionRegistrar",
-			"SolidityFixedFeeRegistrar",
```

Comment:
We still have it under `externalTests/FixedFeeRegistrar`. Not sure why these are disabled though (i.e. failures, slowdowns, or maybe they already run separately?) so I'm not sure if we should update the path or remove it.

BTW, what's the problem with it on Windows? What are we missing that it does not fail in CI?

Reply:
Ah, wait these reference the suites from .cpp files and in #11426 @axic moved this one to a .sol file. So regardless of why these suites are ignored, `SolidityFixedFeeRegistrar` is indeed not needed here.

Reply:
>what's the problem with it on Windows

The problem was in Debug configuration (Windows should not really be relevant indeed), because of the failing `assert`.

Maybe I should put it a way around, and say that the problem was that tests were succeeding in non-Debug configurations because `assert` was disabled there :-).

Reply:
Oh, really? We should definitely have plain asserts enabled, even if we generally avoid them. That can be done separately though: #12641.